modules/light-clients/08-wasm/keeper/msg_server.go

```diff
@@ -31,3 +32,19 @@ func (k Keeper) StoreCode(goCtx context.Context, msg *types.MsgStoreCode) (*type
 		Checksum: codeHash,
 	}, nil
 }
+
+// RemoveCodeHash defines a rpc handler method for MsgRemoveCodeHash
+func (k Keeper) RemoveCodeHash(goCtx context.Context, msg *types.MsgRemoveCodeHash) (*types.MsgRemoveCodeHashResponse, error) {
+	if k.GetAuthority() != msg.Signer {
+		return nil, errorsmod.Wrapf(ibcerrors.ErrUnauthorized, "expected %s, got %s", k.GetAuthority(), msg.Signer)
+	}
+
+	found := types.HasCodeHash(goCtx, msg.CodeHash)
```

Comment:
I though `HasCodeHash` takes an SDK context?

```suggestion
	if !types.HasCodeHash(goCtx, msg.CodeHash) {
		return nil, errorsmod.Wrap(types.ErrWasmCodeHashNotFound, msg.CodeHash)
	}
```

Reply:
I modified it. Collections take goCtx. SDK is planning to get rid of `sdk.Ctx` and use `context.Context` instead

Reply:
isn't @crodriguezvega's suggestion still relevant here? I think an error should be returned `if !types.HasCodeHash()`